install.md

```diff
@@ -138,6 +138,32 @@ sudo apt-get install \
   uidmap
 ```
 
+On Manjaro (and maybe other Linux distributions):
+
+Make sure that the Linux kernel supports user namespaces:
+
+```
+> zgrep CONFIG_USER_NS /proc/config.gz
+CONFIG_USER_NS=y
+
+```
+
+If not, please update the kernel.
+For Manjaro Linux the instructions can be found here:
```

Comment:
I'd prefer `[here](https://wiki.manjaro.org/index.php/Manjaro_Kernels)`
But not enough to hold the PR up, but a touchup at some point in the future.